tests/integration/targets/azure_rm_backuppolicy/tasks/main.yml

```diff
@@ -166,3 +230,4 @@
   ansible.builtin.assert:
     that:
       - not weekly_policy_output_delete_idempotent.changed
+
```

Comment:
Too many blank lines
```suggestion
```

Reply:
ansible-lint suggests you add an new line char at the end https://yamllint.readthedocs.io/en/stable/rules.html#module-yamllint.rules.new_line_at_end_of_file

Reply:
@Klaas- Preferably not, because errors are reported when lint checks.